src/main/java/com/ithaca/book/BookController.java

```diff
@@ -0,0 +1,34 @@
+package com.ithaca.book;
+
+import com.ithaca.group.Book_Group;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+/**
+ * Created by David on 4/25/16.
+ */
+
+@RestController
+@RequestMapping("/books")
+public class BookController {
+
+    @Autowired
+    BookServiceImpl bookService;
+
+    @RequestMapping(method = RequestMethod.POST)
```

Comment:
For the book controller (just to be clear this is not the group stuff which is in the group controller), I have 3 endpoints set up.

1. a post to /books with String:title will give you back the top 10 closest matching Books we have in our database. This is for the user to "search" for the book.

2. a get to /books/{id} where id is the id of the book, returns the actual Book (title, author, other info we have stored)

3. a get to /books/{id}/groups returns a list of all of the groups associated with a particular Book. This is for when the user wants to decide whether to join an existing group or make a new one.